File: 백준/Bronze/10989. 수 정렬하기 3/수 정렬하기 3.java
import java.io.*;
public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int N = Integer.parseInt(br.readLine());
		int[] arr = new int[10001];
		for(int i = 0; i < N; i++) {
			int num = Integer.parseInt(br.readLine());
			arr[num]++;
		}
		
		StringBuilder sb = new StringBuilder();
		for(int i = 0; i < arr.length; i++) {
			for(int j = 0; j < arr[i]; j++) {
				sb.append(i + "\n");
			}
		}
		
		
		System.out.println(sb.toString());
	}
}

File: 백준/Bronze/10989. 수 정렬하기 3/수 정렬하기 3.kt
import java.io.*
fun main() {
    val br = System.`in`.bufferedReader()
    val bw = System.`out`.bufferedWriter()
    val N = Integer.parseInt(br.readLine())
    val arr = Array(10001) {i -> 0}

    for(i in 0 until N) {
        val num = Integer.parseInt(br.readLine());
        arr[num]++
    }
    
    for((i, value) in arr.withIndex()) {
        for(j in 0 until value) {
            bw.write("${i}\n")
        }
    }

    bw.flush()
    bw.close()
}